R/parse_compat.R

```diff
@@ -54,6 +56,22 @@ parse_compat_fix_user <- function(expr, call) {
 }
 
 
+parse_compat_fix_parameter_array <- function(expr, call) {
+  is_parameter_array_assignment <-
+    rlang::is_call(expr$value, c("<-", "=")) &&
+    rlang::is_call(expr$value[[3]], "parameter") &&
+    rlang::is_call(expr$value[[2]], "[")
+
+  if (is_parameter_array_assignment) {
+    original <- expr$valeu
```

Comment:
```suggestion
    original <- expr$value
```

Reply:
This makes no difference as expr$compat is not set, so original is never actually used in the call